packages/react/src/LogoSuite/LogoSuite.features.stories.tsx

```diff
@@ -102,21 +102,13 @@ export const CondensedGap = () => (
       </LogoSuite>
       <LogoSuite>
         <LogoSuite.Heading visuallyHidden>Heading</LogoSuite.Heading>
-        <LogoSuite.Logobar gap="condensed" marquee>
+        <LogoSuite.Logobar gap="condensed" marquee marqueeSpeed="idle">
```

Comment:
Any chance you could add `idle` back in to the `marqueeSpeed` args in the Storybook Controls too please? 🙏

Reply:
yep, nice spot. Feels weird these are so manual though. We should look at importing the options from the main component file at some point to prevent drift.